test/e2e/tokens/erc20.test.mjs

```diff
@@ -305,6 +306,9 @@ describe('ERC20 tests', () => {
       await makeBlock();
 
       userL2BalanceBefore = await getLayer2Balances(nf3User, erc20Address);
+      // Add two transactions in same block to check that finalize withdrawal works
+      // https://github.com/EYBlockchain/nightfall_3/issues/1367
+      await nf3User.deposit(erc20Address, tokenType, transferValue, tokenId, fee);
```

Comment:
If the bug is what I framed above, this is not really doing much as transactions are originated from the same client. In which case I will leave the test as it was without further changes.